src/diffusers/pipelines/controlnet/pipeline_controlnet_inpaint.py

```diff
@@ -1264,6 +1298,13 @@ def __call__(
         else:
             batch_size = prompt_embeds.shape[0]
 
+        if padding_mask_crop is not None:
```

Comment:
um just saw your issue https://github.com/huggingface/diffusers/issues/6435
maybe we need to move this code into `prepare_control_image()`? 

see my comment here https://github.com/huggingface/diffusers/issues/6435#issuecomment-1884345881

Reply:
I don't think it would work. `width` and `height` are still None in there. Do you think we should handle None in `get_crop_region`?

Reply:
ok! 
you can use `self.image_processor. get_default_height_width(image)` to get it